powerspikegg/gateway/BUILD

```diff
@@ -0,0 +1,76 @@
+load("@io_bazel_rules_scala//scala:scala.bzl", "scala_library", "scala_binary", "scala_test")
+
+filegroup(
+    name = "src",
+    srcs = glob([
+        "src/**/*",
+    ]),
+)
+
+scala_binary(
+    name = "server",
+    srcs = ["src"],
+    main_class = "gg.powerspike.Main",
+    deps = [
+        "akka-http_2.11",
+        "grpc-all",
+        "//powerspikegg/rawdata/fetcher:service_java",
+    ],
+)
+
+java_library(
```

Comment:
Add the following TODO:

`TODO(funkysayu): move this to the third_party directory`